services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/enums/PermissionKey.java

```diff
@@ -0,0 +1,57 @@
+package com.sourcefuse.jarc.services.usertenantservice.enums;
+
+public enum PermissionKey {
+    CreateUserGroup("CreateUserGroup", "1"),
+    ViewUserGroupList("ViewUserGroupList", "2"),
+    UpdateUserGroup("UpdateUserGroup", "3"),
+    DeleteUserGroup("DeleteUserGroup", "4"),
+    DeleteSubscriptions("DeleteSubscriptions", "5"),
+    ViewRoles("ViewRoles", "6"),
+    NotAllowed("NotAllowed", "7"),
+    CreateRoles("CreateRoles", "8"),
+    UpdateRoles("UpdateRoles", "9"),
+    DeleteRoles("DeleteRoles", "10"),
+    ViewAnyUser("ViewAnyUser", "11"),
+    ViewTenantUser("ViewTenantUser", "12"),
+    ViewTenantUserRestricted("ViewTenantUserRestricted", "13"),
+    ViewAllUser("ViewAllUser", "14"),
+    ViewOwnUser("ViewOwnUser", "15"),
+    CreateAnyUser("CreateAnyUser", "16"),
+    CreateTenantUser("CreateTenantUser", "17"),
+    CreateTenantUserRestricted("CreateTenantUserRestricted", "18"),
+    UpdateAnyUser("UpdateAnyUser", "19"),
+    UpdateOwnUser("UpdateOwnUser", "20"),
+    UpdateTenantUser("UpdateTenantUser", "21"),
+    UpdateTenantUserRestricted("UpdateTenantUserRestricted", "22"),
```

Comment:
we just need number and not the string as a a param for this and also numbers should not be in string

Reply:
done